doc/_static/style.css

```diff
@@ -199,3 +199,38 @@ a.copybtn {
 .sphx-glr-single-img {
     max-width: 80%!important;
 }
+
+.team-column {
+    float: left;
+    width: 20%;
```

Comment:
Can we add some spaces between columns? 

I don't know much about CSS, but the following changes look better to me:
```suggestion
    width: 18%;
    margin: 0.25em;
```

**Current**:

![image](https://user-images.githubusercontent.com/3974108/120080771-69520400-c088-11eb-9a10-5e75b47d14ef.png)


**After**
![image](https://user-images.githubusercontent.com/3974108/120080740-4aec0880-c088-11eb-9a68-c9b8c8b7f66c.png)

Reply:
Looks good.